prsa/quantification.py

```diff
@@ -0,0 +1,155 @@
+import numpy as np
+
+"""
+Source for the following two formulas:
+Chen T, Feng G, Heiselman C, Quirk JG, Djurić PM. IMPROVING PHASE-RECTIFIED SIGNAL 
+AVERAGING FOR FETAL HEART RATE ANALYSIS. Proc IEEE Int Conf Acoust Speech Signal Process. 
+2022;2022:10.1109/icassp43922.2022.9747860. doi:10.1109/icassp43922.2022.9747860
+"""
+
+def calculate_ac(prsa_output_ac: np.ndarray, L: int, s: int) -> float:
```

Comment:
Formula in line 15 matches formula from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9415860/ and due to it should be the same for dc in line 41(but isn't so two seperate functions shouldn't be needed), but actual code in lines 30-31 and 56-57 isn't the same comparing them to themselfs and also to the article and to the documentation